frontend/src/Pages/Login.js

```diff
@@ -168,9 +169,8 @@ export default function Login() {
 			password: password
 		})
 		.then((resp) => {
-			const userId = resp.data.user_id;
-			console.log(resp.data);
 			authenticateWithOpenId(resp.data);
```

Comment:
This isn't awaited, so you'll be immediately navigated without waiting for the response.

Reply:
I attached a then call to the returned promise instead. Does that work?